### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by gaabral**
Changes: added search bar in UI.
Please review: @phmp @LordVader14

### CODE REVIEW

File: src/main/webapp/resources/css/mystyle.css [hunk 1]
```diff
@@ -51,3 +51,64 @@
...
+    position: relative;
+    background-color: #ffffff;
+    border: 1px solid #bdbdbd;
+    box-shadow: 0 1px 3px 0 #bdbdbd;
+    border-radius: 4px;
+    margin: 0 auto;
+    display: table;
+    width: 100%;
+}
+.search-bar .search-bar-location,
```
Comment (phmp) [inline src/main/webapp/resources/css/mystyle.css]: What do you think about one class for these elements?

File: src/main/webapp/resources/html/index.html [hunk 2]
```diff
@@ -15,6 +15,26 @@
...
 
+<!--Search bar definition -->
+<form  name="Search" action="/search" method="get">
+<div class="search-bar">
+    <div class="search-bar-row ">
+        <div class="search-bar-location">
+            <input type="text" class="search-bar-input" name="searchLocation" placeholder="Miasto, ulica">
+        </div>
+        <div class="search-bar-radius">
+            <input type="text" class="search-bar-input" name="searchRadius" placeholder="+0m">
```
Comment (phmp) [inline src/main/webapp/resources/html/index.html:26]: Close input /

**Comment by LordVader14** [comment]
Hi, looks fine. There is one problem with the "Szukaj" button (it becomes blue when mouse is over it, but I think we can leave it for now). One question @gaabral @phmp : what do you think is better: having one big .css file or multiple ones?

File: src/main/webapp/resources/css/mystyle.css [hunk 3]
```diff
@@ -51,3 +51,64 @@
...
+.search-bar .search-bar-radius,
```
Comment (LordVader14) [inline src/main/webapp/resources/css/mystyle.css]: you can remove .search-bar (leave only .search-bar-radius etc), like:
.search-bar-location,
.search-bar-radius,
.search-bar-date,
.search-bar-submit {
    display: table-cell;
    text-align: center;
    vertical-align: middle;
    padding: 10px 16px 10px;
}

File: src/main/webapp/resources/html/index.html (hunk 2 above)
Comment (gaabral) [reply src/main/webapp/resources/html/index.html:26]: In HTML, the < input > tag has no end tag - if this is what you mean here

Comment (phmp) [reply src/main/webapp/resources/html/index.html:26]: Ok. thanks

**Comment by gaabral** [comment]
I aggregated search barr elements into one common class. I'm not changing button colors for now, as I can't find the requirements on mouse hover, on click and after click.

**Review by phmp — APPROVED** [review]
for me looks fine

**Comment by gaabral** [comment]
@LordVader14 I think we should have one css file for one whole view, describing all elements, and if we want another view, lets say in different color scheme, then separate css file, again, with all elements defined. 
So for now I would leave only one css file as we have only one color scheme.

**Pull request merged**